### PR TITLE
Move all components to tailwind (remove next-ui)

### PR DESCRIPTION
**Pull request by hamidra**
This moves all dependencies to next-ui and only relies on tailwind to style all components.

### CODE REVIEW

File: assets/main.css [hunk 1]
```diff
@@ -21,12 +21,17 @@
 }
 
 html {
-  font-family: Unbounded, Helvetica Neue, Helvetica, Arial, sans-serif;
+  font-family: Inter, Helvetica Neue, Helvetica, Arial, sans-serif;
```
Comment (jeluard) [inline assets/main.css:24]: Any reason to move away from Unbounded?

File: assets/main.css [hunk 2]
```diff
@@ -21,12 +21,17 @@
...
   font-style: normal;
   font-weight: 400;
   height: 100%;
 }
 
+/* prettier-ignore */
+.font-brand .h1, .h2, .h3, .h4, .h5, h1, h2, h3, h4, h5 {
+  font-family: Unbounded, Helvetica Neue, Helvetica, Arial, sans-serif;
```
Comment (hamidra) [inline assets/main.css:32]: unbounded is used for brand and headers.

File: src/index.html [hunk 3]
```diff
@@ -16,6 +16,22 @@
...
+
+    <!--ToDo:download the fonts to self host-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <script>
```
Comment (jeluard) [inline src/index.html:28]: I removed this script block recently

**Review by jeluard — APPROVED** [review]

**Comment by jeluard** [comment]
@hamidra Looks like you'll need to rebase :/

**Pull request merged**